### PR TITLE
Got the docx download feature working

### PR DESCRIPTION
**Pull request by smcmurtry**

### CODE REVIEW

File: routes/agreement-1/agreement-1.njk [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 {% block content %}
 
+<a href={{ "documents/" + docxFilename }}>Dowload .docx</a>
```
Comment (sastels) [inline routes/agreement-1/agreement-1.njk]: if we put the files in the `/tmp` dir then we wouldn't have to ever worry about deleting them ourselves - maybe worth doing?

Comment (smcmurtry) [reply routes/agreement-1/agreement-1.njk]: I was thinking about that - would I just create one in the public folder, like: `simplify-privacy-statements-V2/public/tmp`? And how does it work, does any folder with that name get automatically cleaned whenever the server restarts? Or does the folder have to be in a certain place?

Comment (sastels) [reply routes/agreement-1/agreement-1.njk]: Ah, I meant the global `/tmp` folder that alpine uses.

Comment (smcmurtry) [reply routes/agreement-1/agreement-1.njk]: In that case I would have to add a route for that - I'll give it a try in a future PR

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,8 @@
-FROM node:10-alpine
+FROM pandoc/core
 LABEL maintainer="-"
 
+RUN apk add --update nodejs nodejs-npm
```
Comment (sastels) [inline Dockerfile]: nice!

**Pull request merged**